index.html

```diff
@@ -79,6 +79,246 @@ <h1>
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+
+            <ul class="social-icons">
+              <li>
+                <a href="twitter.com/" target="_blank">
+                  <img src="./images/twitter.svg" alt="twitter-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="linkedin.com" target="_blank">
+                  <img src="./images/linkedIn.svg" alt="linkedin-icon"
+                /></a>
+              </li>
+
+              <li>
+                <a href="medium.com" target="_blank">
+                  <img src="./images/medium.svg" alt="medium-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="github.com" target="_blank">
+                  <img src="./images/github.svg" alt="github-icon" />
+                </a>
+              </li>
+
+              <li>
+                <a href="angel.com" target="_blank">
+                  <img src="./images/angel.svg" alt="angel-icon" />
+                </a>
+              </li>
+            </ul>
+            <div>
+              <button type="button" class="primary-btn">Get my resume</button>
+            </div>
+
+            <div class="languages-section">
+              <ul class="languages-title">
+                <li>
+                  <h3>Languages</h3>
+                  <a href="#"
+                    ><img src="./images/Arrow-down.svg" alt="arrow-down"
+                  /></a>
+                </li>
+              </ul>
+
+              <ul class="languages-box">
+                <li>
+                  <img src="./images/js-icon.svg" alt="js-icon" />
+                  <a href="#"><p>Javascript</p></a>
+                </li>
+              </ul>
+
+              <ul class="languages-box">
+                <li>
+                  <img src="./images/html-icon.svg" alt="html-icon" />
+                  <a href="#"><p>HTML</p></a>
+                </li>
+              </ul>
+
+              <ul class="languages-box">
+                <li>
+                  <img src="./images/css-icon.svg" alt="css-icon" />
+                  <a href="#"><p>CSS</p></a>
+                </li>
+              </ul>
+
+              <ul class="languages-title">
+                <li>
+                  <h3>Frameworks</h3>
+                  <a href="#"
+                    ><img src="./images/arrow-side.svg" alt="arrow-side"
+                  /></a>
+                </li>
+              </ul>
+              <hr />
+              <ul class="languages-title">
+                <li>
+                  <h3>Skills</h3>
+                  <a href="#"
+                    ><img src="./images/arrow-side.svg" alt="arrow-side"
+                  /></a>
+                </li>
+              </ul>
+              <hr />
```

Comment:
:red_circle:  The bottom line should set with CSS rather than `<hr> `element. Please make sure you set the bottom line using CSS `border-bottom ` rule. You should use something similar to this :arrow_down: 

`border-bottom: 1px solid #enter_code_color_from_figma_here`;

Your Design
![noborder](https://user-images.githubusercontent.com/26736582/198399469-607ede86-1a75-4e4b-874d-79d5dd151739.png)

Figma Design
![borderfigma](https://user-images.githubusercontent.com/26736582/198399501-11b8c4f3-3db1-4d11-b43c-a68ed091fdde.png)